From 4168e825fda016f75e2c5fa24264417dcd39df57 Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Thu, 25 Dec 2014 12:52:13 -0500
Subject: [ethernet] mlx5: Add more supported devices

Message-id: <264a567b9c7baff6eb676079fed7f0ac252b95bb.1419511898.git.avadai@redhat.com>
Patchwork-id: 102404
O-Subject: [Patch RHEL7.1] net/mlx5_core: Add more supported devices
Bugzilla: 1169277
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Eli Cohen <eli@dev.mellanox.co.il>

Add ConnectX-4LX to the list of supported devices as well as their virtual
functions.

Signed-off-by: Eli Cohen <eli@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 28c167fa8f8ea1850e6053bffb8ee30c1ac4411a)

Bugzilla: 1169277
Brew: 8440369
Tested in Mellanox lab
Signed-off-by: Amir Vadai <avadai@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/mellanox/mlx5/core/main.c b/drivers/net/ethernet/mellanox/mlx5/core/main.c
index 98e874e..9fb82fe 100644
--- a/drivers/net/ethernet/mellanox/mlx5/core/main.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/main.c
@@ -781,8 +781,12 @@ static void remove_one(struct pci_dev *pdev)
 }
 
 static const struct pci_device_id mlx5_core_pci_table[] = {
- { PCI_VDEVICE(MELLANOX, 4113) }, /* MT4113 Connect-IB */
+ { PCI_VDEVICE(MELLANOX, 4113) }, /* Connect-IB */
+ { PCI_VDEVICE(MELLANOX, 4114) }, /* Connect-IB VF */
  { PCI_VDEVICE(MELLANOX, 4115) }, /* ConnectX-4 */
+ { PCI_VDEVICE(MELLANOX, 4116) }, /* ConnectX-4 VF */
+ { PCI_VDEVICE(MELLANOX, 4117) }, /* ConnectX-4LX */
+ { PCI_VDEVICE(MELLANOX, 4118) }, /* ConnectX-4LX VF */
  { 0, }
 };
 
-- 
1.7.1